cli/azd/cmd/templates.go

```diff
@@ -27,6 +27,7 @@ func templatesCmd(rootOptions *internal.GlobalCommandOptions) *cobra.Command {
 		[]output.Format{output.JsonFormat, output.TableFormat},
 		output.TableFormat,
 	))
+
```

Comment:
nit: remove newline?

Reply:
My formatter added it.. Code files should end with an empty new line.